client/bindings/python/Cargo.toml

```diff
@@ -1,7 +1,5 @@
-[workspace]
-
 [package]
-name = "iota-client"
+name = "iota-client-python"
```

Comment:
This name is currently used for the binding
But looks like it's possible to specify another name using
```
[package.metadata.maturin]
name = "othername"
```

Reply:
Used by what exactly?

Reply:
For the python package name

Reply:
Done